int64 flg
---
float64 treasure_x
float64 treasure_y